commands/deploy_test.go

```diff
@@ -83,7 +83,12 @@ func Test_deploy(t *testing.T) {
 			ResponseStatusCode: http.StatusOK,
 		},
 	})
-	defer s.Close()
+
+	defer func() {
+		gateway = defaultGateway
```

Comment:
This feels odd, cleared to initialise vars within the tests for clarity rather than having this outside? Could you point to why this is needed by the way?